### PR TITLE
Add profiles section.

### PR DESCRIPTION
**Pull request by KonovalovaVV**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -166,4 +166,39 @@
...
+    align-items: flex-start;
+}
+
+.profile__text {
+    display: flex;
+    flex-direction: column;
+    align-content: space-between;
+}
+
+.profile__text-description {
```
Comment (MaksHladki) [inline css/main.css:191]: Rename to `profile__description`

File: index.html [hunk 2]
```diff
@@ -165,9 +165,54 @@ <h2 class="header-label header-label--dark">Cras ornare tristique elit</h2>
...
-    </div>
-
+    <section class="section">
+      <div class="profile">
+        <div class="profile__item">
+          <img class="profile__image" src="img/profile1.png" alt="">
+          <div class="profile__text">
+            <h3 class="lead">Integer vitae libero</h3>
+            <p class="profile__text-description description">
+              Praesent dapibus, neque id cursus faucibus, tortor neque egestas <br />
```
Comment (MaksHladki) [inline index.html:175]: Please, remove br tags from the description.

File: index.html [hunk 3]
```diff
@@ -165,9 +165,54 @@ <h2 class="header-label header-label--dark">Cras ornare tristique elit</h2>
...
+              augue, eu vulputate magna eros eu erat. Aliquam erat volutpat. Nam <br />
```
Comment (MaksHladki) [inline index.html:176]: Please, re-check the line-height.
image

File: index.html [hunk 4]
```diff
@@ -165,9 +165,54 @@ <h2 class="header-label header-label--dark">Cras ornare tristique elit</h2>
...
+              dui mi, tincidunt lorem ipsuk dolor <a href="#" class="more-link">+ more</a>
```
Comment (MaksHladki) [inline index.html:177]: Please, fix a long text problem 
image

File: css/main.css (hunk 1 above)
Comment (KonovalovaVV) [reply css/main.css:191]: Ok.

File: index.html (hunk 3 above)
Comment (KonovalovaVV) [reply index.html:176]: Ok.

File: index.html (hunk 2 above)
Comment (KonovalovaVV) [reply index.html:175]: Ok.

**Pull request merged**